LibraBFT/Impl/Consensus/RoundManager.agda

```diff
@@ -92,14 +86,14 @@ module ensureRoundAndSyncUpM
 
   step₀ = do
     currentRound ← use (lRoundState ∙ rsCurrentRound)
-    if ⌊ messageRound <? currentRound ⌋
+    ifM messageRound <? currentRound
```

Comment:
I'm definitely confused.

In the haskell code, the `M` suffix was generally used to mean functions in the `LBFT` monad.

This particular `ifM` seems that it is turning the decidable test into a boolean.   That has nothing to do with a monad nor a `LBFT` monad.